examples/discordjs/package.json

```diff
@@ -1,15 +1,18 @@
 {
-  "name": "discordjs",
-  "version": "1.0.0",
+  "name": "discordjs_starter",
+  "version": "1.0.2",
   "main": "src/index.js",
-  "author": "Jake Runzer",
+  "author": "Korab Arifi",
```

Comment:
Generally would advise against changing the author... but alright.

Reply:
oops